modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -84,6 +85,35 @@ def remove_words(hash, message)
         end
         message
       end
+
+      def attachments
```

Comment:
Do you see this living in controller long term?

Reply:
Great call! I refactored this out. I think it's better this way. What do you think?